dotCMS/src/main/resources/dotmarketing-config.properties

```diff
@@ -861,3 +861,4 @@ STARTER_BUILD_VERSION=${starter.deploy.version}
 ##LTS properties to show EOL message
 show.lts.eol.message=false
 date.lts.eol=12/31/2099
+
```

Comment:
remove this file from the diff

Reply:
done https://github.com/dotCMS/core/pull/30466/commits/f2996ecf0f05da6ccf414ba9fc6bd851a4099679